ColorSupplier.java

```diff
@@ -0,0 +1,9 @@
+import java.util.Random;
+public class ColorSupplier {
+    public Colors getRandomColor() {
+        Random random = new Random();
+        Colors[] color = Colors.values();
+        int gege = random.nextInt(color.length);

```

Comment:
The variable name `gege` is not descriptive. Consider renaming it to something more meaningful, like `randomIndex`, to improve code readability.